resources/public/api/conf/2.0/application.raml

```diff
@@ -0,0 +1,74 @@
+#%RAML 1.0
```

Comment:
Is this line just a comment or does it have special meaning?

Reply:
It marks the document as a RAML file.  A bit like the shebang in Bash